tests/foreman/cli/test_computeresource_libvirt.py

```diff
@@ -106,136 +106,71 @@ def libvirt_url():
     return LIBVIRT_RESOURCE_URL % settings.libvirt.libvirt_hostname
 
 
-@pytest.mark.tier1
-def test_positive_create_with_name(libvirt_url, module_target_sat):
-    """Create Compute Resource
+@pytest.mark.e2e
```

Comment:
We should not mark it e2e as it is just a crud

Reply:
we mark all CR crud tests as e2e, along with e2e provisoning tests

Reply:
This was earlier discussed that having crud for compute resource tests is not enough to make it e2e.

Reply:
@shweta83 we do have similar tests marked as e2e in vmware, libvirt, gce across ui and api as well